pulser/_seq_drawer.py

```diff
@@ -74,11 +74,16 @@ def gather_data(seq):
     return data
 
 
-def draw_sequence(seq):
+def draw_sequence(seq, time_slices=np.array([])):
```

Comment:
It's a bit weird to put an empty array as a default. I'd rather have `None`. Then you can switch the `if len(time_slices) > 0` by just `if time_slices`

Reply:
that was my first version, but it felt wrong somehow. I can definitely change it back though, less code is always better